src/InjectHostDevBufferCopies.cpp

```diff
@@ -129,7 +129,7 @@ class InjectBufferCopies : public IRMutator {
     using IRMutator::visit;
 
     // BufferInfo tracks the state of a givven buffer over an IR scope.
-    // Generally the scope is a ProducerConsumer. The data herein is mutable.
+    // Generally the scope is a Producer/Consumer. The data herein is mutable.
```

Comment:
What does "Generally the scope is a Producer/Consumer." mean? I am having trouble picture the requirement on the IR here.